src/main/java/org/iac2/service/checking/plugin/implementation/subgraphmatching/comparison/ComponentComparisonResult.java

```diff
@@ -0,0 +1,4 @@
+package org.iac2.service.checking.plugin.implementation.subgraphmatching.comparison;
+
+public record ComponentComparisonResult(ComponentComparisonOutcome outcome, String propertyName, String expression) {
```

Comment:
record? what?

https://www.geeksforgeeks.org/what-are-java-records-and-how-to-use-them-alongside-constructors-and-methods/ 
I'm getting old <.<

Reply:
I didn't know about them either didn't know about them either, but IntelliJ was really persistent about converting certain classes to records... so I agreed :D